octane/evmengine/keeper/db.go

```diff
@@ -160,5 +161,28 @@ func (k *Keeper) EligibleWithdrawals(ctx context.Context) ([]*Withdrawal, error)
 		}
 	}
 
-	return withdrawals, nil
+	evmWithdrawals := []*etypes.Withdrawal{}
```

Comment:
add a comment that nil withdrawals isn't supported, so using empty slice by default